app/Http/Controllers/ClassroomController.php

```diff
@@ -37,12 +39,14 @@ public function store(Request $request)
         // $data = request()->all();
         $className=request()->className;
```

Comment:
no need to use `request()`, why  not using `$request`, the function receives this